src/main/java/com/google/sps/WebSocketHandler.java

```diff
@@ -3,17 +3,18 @@
 import java.util.HashMap;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Collections;
 
 import javax.websocket.Session;
 
 public class WebSocketHandler {
 
     private static WebSocketHandler instance;
 
-    private HashMap<String, List<Session>> chatRoomMap;
+    private HashMap<Long, List<Session>> chatRoomMap;
```

Comment:
nit: Add a comment about what the Long refers to. Like is it userId, mapId, roomId, or something else?